wso2cep/4.0.0/samples/cartridges/ec2/wso2cep-400/wso2cep-400-presenter.json

```diff
@@ -70,6 +70,10 @@
             "value": "root"
         },
         {
+            "name": "payload_parameter.CONFIG_PARAM_REGISTRY_DB_DRIVER",
+            "value": "com.mysql.jdbc.Driver"
+        },
+        {
```

Comment:
I have added it here because sometimes we comment out driver param in module.ini file. Fixed in commit 13c4c52d547cb5046c828b28b68923d0672fb1d4.